<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inventory Management System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="filterLayout">
      <item>
       <widget class="QLineEdit" name="searchInput">
        <property name="placeholderText">
         <string>Search by Item Name...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="categoryFilter"/>
      </item>
      <item>
       <widget class="QComboBox" name="supplierFilter"/>
      </item>
      <item>
       <widget class="QPushButton" name="searchButton">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="inventoryTable">
      <property name="columnCount">
       <number>6</number>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Category</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Supplier</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Quantity</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_name">
        <property name="text">
         <string>Item Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nameInput"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_category">
        <property name="text">
         <string>Category:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="categoryCombo"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_supplier">
        <property name="text">
         <string>Supplier:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="supplierCombo"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_quantity">
        <property name="text">
         <string>Quantity:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="quantitySpin">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_price">
        <property name="text">
         <string>Price:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="priceSpin">
        <property name="minimum">
         <double>0.00</double>
        </property>
        <property name="maximum">
         <double>100000.00</double>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateButton">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
